stubs/stripe/stripe/webhook.pyi

```diff
@@ -1,13 +1,15 @@
-from typing import Any
-
-from stripe import error as error
+from stripe import Event, error as error
 
 class Webhook:
     DEFAULT_TOLERANCE: int
     @staticmethod
-    def construct_event(payload, sig_header, secret, tolerance=..., api_key: Any | None = ...): ...
+    def construct_event(
+        payload: bytes | str, sig_header: str, secret: str, tolerance: int = ..., api_key: str | None = ...
+    ) -> Event: ...
 
 class WebhookSignature:
     EXPECTED_SCHEME: str
     @classmethod
-    def verify_header(cls, payload, header, secret, tolerance: Any | None = ...): ...
+    def verify_header(cls, payload: bytes | str, header: str, secret: str, tolerance: int | None = ...) -> bool: ...
```

Comment:
It actually only ever returns `True` (you'll need to import `Literal` from `typing_extensions`)

```suggestion
    def verify_header(cls, payload: bytes | str, header: str, secret: str, tolerance: int | None = ...) -> Literal[True]: ...
```